ci/test_python.sh

```diff
@@ -210,14 +210,9 @@ if [[ "${RAPIDS_CUDA_VERSION}" == "11.8.0" ]]; then
       --channel "${PYTHON_CHANNEL}" \
       --channel pyg \
       "cugraph-pyg" \
+      "pytorch>=2.3,<2.4"
```

Comment:
```suggestion
      "pytorch>=2.3,<2.4" \
```

Without this trailing slash, this will miss the `ogb` line